api_server/swagger/firecracker-experimental.yaml

```diff
@@ -354,38 +354,38 @@ paths:
           schema:
             $ref: "#/definitions/Error"
 
-    /vsocks/{id}:
-      put:
-        summary: Creates new vsock with ID specified by the id parameter.
-        description:
-          If the vsock device with the specified ID already exists, its body will
-          be updated based on the new input. May fail if update is not possible.
-        operationId: putGuestVsockByID
-        parameters:
-        - name: id
-          in: path
-          description: The id of the vsock device
-          required: true
-          type: string
-        - name: body
-          in: body
-          description: Guest vsock properties
-          required: true
-          schema:
-            $ref: "#/definitions/Vsock"
-        responses:
-          201:
-            description: Vsock created
-          204:
-            description: Vsock updated
-          400:
-            description: Vsock cannot be created due to bad input
-            schema:
-              $ref: "#/definitions/Error"
-          default:
-            description: Internal server error
-            schema:
-              $ref: "#/definitions/Error"
+  /vsocks/{id}:
```

Comment:
Please also add an entry about this fix to the CHANGELOG.md file.

Reply:
Done